status_deprecated.go

```diff
@@ -64,7 +63,7 @@ func FetchStatusDeprecated(f *Ferry, v Verifier) *StatusDeprecated {
 	status.SourceHostPort = fmt.Sprintf("%s:%d", f.Source.Host, f.Source.Port)
 	status.TargetHostPort = fmt.Sprintf("%s:%d", f.Target.Host, f.Target.Port)
 
-	status.OverallState.Store(f.OverallState.Load())
+	status.OverallState = fmt.Sprintf("%s", f.OverallState.Load())
```

Comment:
Alternatively, I haven't verified it myself but I think something like this one-line change will work too: https://github.com/Shopify/ghostferry/blob/1ad39c7ab99f58c2d5858e01f0ab734e789ddeff/ferry.go#L798.